docs/telemetry.md

```diff
@@ -0,0 +1,43 @@
+# Telemetry on Percona XtraDB Cluster
+
+Percona telemetry fills in the gaps in our understanding of how you use Percona XtraDB Cluster. Participation in the anonymous program is optional. You can opt-out if you prefer not to share information.
+
+Read our [Percona Privacy statement] to learn more.
+
+## What information is collected
+
+Currently, Percona XtraDB Cluster only collects information about the installation environment. Future releases may add additional metrics.
+
+Be assured that the access to this raw data is rigorously controlled, and user identification within the dataset is impossible. The data is thoroughly anonymized and cannot be traced to any specific user.
+
+## Disable telemetry
+
+Starting with Percona XtraDB Cluster 8.0.34, the telemetry is enabled by default. If you decide to not send usage data to Percona, you can set the `PERCONA_TELEMETRY_DISABLE=1` environment variable for either the root user or in the operating system prior to the installation process.
+
+=== "Debian-derived distribution"
+
+    Add the environment variable before the install process.
+
+    ```{.bash data-prompt="$"}
+    $ sudo PERCONA_TELEMETRY_DISABLE=1 apt install percona-server-server
+    ```
+
+=== "Red Hat-derived distribution"
+
+    Add the environment variable before the install process.
+    
+    ```{.bash data-prompt="$"}
+    $ sudo PERCONA_TELEMETRY_DISABLE=1 yum install percona-server-server
+    ```
+
+=== "DOCKER"
+
+    Add the environment variable when running a command in a new container.
+    
+    ```{.bash data-prompt="$"}
+    $ docker run percona/percona-server -e PERCONA_TELEMETRY_DISABLE=1
```

Comment:
I would use command that's more related to cluster:
docker run -d \
  -e MYSQL_ROOT_PASSWORD=test1234# \
 -e PERCONA_TELEMETRY_DISABLE=1 \
  -e CLUSTER_NAME=pxc-cluster1 \
  --name=pxc-node1 \
  percona/percona-xtradb-cluster:8.0